### ISSUE TITLE
New Funktionality to depend on several fields at the same time

### ISSUE THREAD

**Issue by Sergej-Tihonov** [orlyapps/nova-belongsto-depend#81]
Pull request for this is on the way.
We did it backward compatible from the syntax.

**Comment by MeerKatDev** [comment]
any news on this?

**Comment by cwilby** [comment]
@MeerKatDev PR here - https://github.com/orlyapps/nova-belongsto-depend/pull/82

**Comment by Sergej-Tihonov** [comment]
Short update on this topic.
I failed a bit in the last PR, overlooked a part of the implementation.
But have a new PR open:
https://github.com/orlyapps/nova-belongsto-depend/pull/91
Here the problem is fixed.
Added some documentation about it as well.

Additionally I have added the instanceof array check with exception throw if dependsOn is not an array.
Thank you for that hint, @cwilby.

And your question about this syntax:
`public function dependsOn(string ...$classNames)`
This makes $className always to an array. With one or several elements passed in.
